stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/viewholder/PlainTextWithFileAttachmentsViewHolder.kt

```diff
@@ -7,38 +7,56 @@ import io.getstream.chat.android.ui.databinding.StreamUiItemMessagePlainTextWith
 import io.getstream.chat.android.ui.messages.adapter.BaseMessageItemViewHolder
 import io.getstream.chat.android.ui.messages.adapter.MessageListItemPayloadDiff
 import io.getstream.chat.android.ui.messages.adapter.MessageListListenerContainer
+import io.getstream.chat.android.ui.messages.adapter.view.AttachmentClickListener
+import io.getstream.chat.android.ui.messages.adapter.view.AttachmentLongClickListener
 import io.getstream.chat.android.ui.messages.adapter.viewholder.decorator.Decorator
 import io.getstream.chat.android.ui.utils.extensions.hasLink
 
 public class PlainTextWithFileAttachmentsViewHolder(
     parent: ViewGroup,
     decorators: List<Decorator>,
-    private val listenerContainer: MessageListListenerContainer?,
-    internal val binding: StreamUiItemMessagePlainTextWithFileAttachmentsBinding = StreamUiItemMessagePlainTextWithFileAttachmentsBinding.inflate(
-        parent.inflater,
-        parent,
-        false
-    )
+    listenerContainer: MessageListListenerContainer?,
+    internal val binding: StreamUiItemMessagePlainTextWithFileAttachmentsBinding =
+        StreamUiItemMessagePlainTextWithFileAttachmentsBinding.inflate(
+            parent.inflater,
+            parent,
+            false
+        )
 ) : BaseMessageItemViewHolder<MessageListItem.MessageItem>(binding.root, decorators) {
 
-    public override fun bindData(data: MessageListItem.MessageItem, diff: MessageListItemPayloadDiff?) {
-        binding.messageText.text = data.message.text
+    init {
+        listenerContainer?.let { listeners ->
+            binding.run {
```

Comment:
Thanks for making these consistent, I didn't want to tack on this change onto one of my own PRs 👍